documentation/docs/getting-started/using-extensions.md

```diff
@@ -246,6 +246,57 @@ Note: Java and Kotlin extensions are only support on Linux and macOS
   </TabItem>
 </Tabs>
 
+
+### Deeplinks
+
+Deeplinks provide a convenient way to install and configure remote extensions in Goose Desktop. They use a `goose://` URL scheme with an `id`, `name` and `description` argument, and either a `url` argument or `cmd` and `arg` arguments.
```

Comment:
feel like we can keep this part succinct because we will explain the different components of the URL after this, so it's a little redundant imo. 

```suggestion
Extensions can be installed using Goose's deep link protocol. The URL format varies based on the extension type:
```